backend/src/services/secret/secret-service.ts

```diff
@@ -998,6 +1000,9 @@ export const secretServiceFactory = ({
     });
 
     const decryptedSecrets = secrets.map((el) => decryptSecretRaw(el, botKey));
+    const filteredSecrets = tagSlugs.length
```

Comment:
question: why not add this filter to the sql query?

Reply:
The reasoning was to filter by name and slug but i guess it's not needed anymore. 

But i guess with unification of this then it should be ok

Reply:
Keeping it for now to reduce complexity as changing the filter in secret dal would cause a lot of changes in multiple places at the moment. Will address this on new secret service